docs/design/README.md

```diff
@@ -3053,18 +3180,18 @@ Carbon generics have a number of other features, including:
     same data representation as an existing type, so you may cast between the
     two types, but can implement different interfaces or implement interfaces
     differently.
--   Additional requirements can be placed on the associated types of an
+-   Additional requirements can be placed on the associated constants of an
     interface using
     [`where` constraints](generics/details.md#where-constraints).
 -   [Implied constraints](generics/details.md#implied-constraints) allows some
```

Comment:
```suggestion
-   [Implied constraints](generics/details.md#implied-constraints) allow some
```
Not sure about this: is "implied constraints" a singular noun, short for "the implied constraints feature" or "the use of implied constraints" or similar, or the plural of "implied constraint"?